Validator/Constraints/EmbeddedAddressFormatConstraint.php

```diff
@@ -10,13 +11,21 @@
  *
  * @codeCoverageIgnore
  */
+#[\Attribute(\Attribute::TARGET_PROPERTY)]
 class EmbeddedAddressFormatConstraint extends AddressFormatConstraint
 {
-    /**
-     * {@inheritdoc}
-     */
-    public function getTargets()
-    {
-        return self::PROPERTY_CONSTRAINT;
-    }
+  public function __construct($options = null, $fieldOverrides = [])
+  {
+    $this->fieldOverrides = new FieldOverrides($fieldOverrides);
```

Comment:
Why is this change here? This is not related to PHP8 compatibility. Please remove this. AFAICS the constructor in here is not necessary, since there are no local variables.